Emby.Naming/Common/NamingOptions.cs

```diff
@@ -311,6 +311,12 @@ public NamingOptions()
                     }
                 },
 
+                // "Foo Bar 889"
+                new EpisodeExpression(@".*[\\\/](?![Ee]pisode)(?<seriesname>(\w+\s*?)*)\s(?<epnumber>\d{1,3})(-(?<endingepnumber>\d{2,3}))*[^\\\/]*$")
```

Comment:
Is this a parsing fix or a new naming rule? Looks like the latter to me.

Reply:
You should also move this below the comment about Kodi rules if this isn't directly from Kodi.

Reply:
There is a test for this rule, so...
And one of the kodi rules gives a false positive, that’s why it’s before that one.

Reply:
What would a false positive look like for a naming rule? Wouldn't that just be covered by the Kodi rule, or would it not always work consistently?

Reply:
It gave back 8 as season number, while there was none.

Reply:
I would add that as a comment then, just in case this has further edge cases to account for in the future.